.github/workflows/master.yml

```diff
@@ -81,8 +81,6 @@ jobs:
           python -m venv .venv
           source .venv/bin/activate
           source ./dev/install-common-deps.sh --ml
-          # SQLAlchemy==2.0.33 has a regression: https://github.com/sqlalchemy/sqlalchemy/issues/11832
-          pip install SQLAlchemy!=2.0.33
```

Comment:
There is still a chance that 2.0.33 is installed if there is a package that has `SQLAlchemy==2.0.33`, it's unlikely to happen.